sandbox/grommet-app/src/pages/layouts/kinds/OpsRamp/ResourceAlerts.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { Box, Text } from 'grommet';
+import { Metric, metricSizes } from '../../../../components';
+import {
+  StatusGoodSmall,
+  StatusCriticalSmall,
+  StatusWarningSmall,
+  StatusInfo,
+  StatusPlaceholderSmall,
+} from 'grommet-icons';
+
+interface AlertItemProps {
+  label: string;
+  Icon: React.ComponentType<{ color?: string }>;
+  color: string;
+  value: number;
+  size: string;
+}
+
+const AlertItem: React.FC<AlertItemProps> = ({
+  label,
+  Icon,
+  color,
+  value,
+  size,
+}) => (
+  <AlertMetric
+    size={size}
+    label={label}
+    value={
+      <Box direction="row" align="center" gap="xsmall">
+        <Icon color={color} />
+        <Text size={metricSizes[size].value} color="text-strong" weight={500}>
+          {value}
+        </Text>
+      </Box>
+    }
+  />
+);
+
+interface AlertMetricProps {
+  label: string;
+  size: string;
+  value: JSX.Element;
+}
+
+const AlertMetric: React.FC<AlertMetricProps> = ({ label, size, value }) => (
+  <Metric
+    align="center"
+    label={label}
+    value={value}
+    unit={null}
+    options={{}}
+    reverse
+    size={size}
+  />
+);
+
+const TicketNotesSummary: React.FC = () => (
+  <Box pad={{ vertical: 'small' }} direction="row" gap="small">
+    <Text>
+      Tickets: <Text weight="bold">1</Text>
+    </Text>
+    <Text>
+      Notes: <Text weight="bold">2</Text>
+    </Text>
+  </Box>
+);
+
+interface ResourceAlertsProps {
+  size: string;
+}
+
+export const ResourceAlerts: React.FC<ResourceAlertsProps> = ({ size }) => {
+  const alertData = [
+    { label: 'All', Icon: StatusGoodSmall, color: 'status-ok', value: 12 },
+    {
+      label: 'Critical',
+      Icon: StatusCriticalSmall,
+      color: 'status-critical',
+      value: 4,
+    },
+    {
+      label: 'Warning',
+      Icon: StatusWarningSmall,
+      color: 'status-warning',
+      value: 2,
+    },
+    { label: 'OK', Icon: StatusGoodSmall, color: 'status-ok', value: 0 },
+    { label: 'Info', Icon: StatusInfo, value: 4 },
+    {
+      label: 'Observed',
+      Icon: StatusPlaceholderSmall,
+      value: 2,
+    },
+  ];
+
+  return (
+    <Box>
+      <Text>Resource Alerts</Text>
+      {/* // probably need a grid for responsive layout */}
+      <Box gap="small" direction="row">
```

Comment:
It might be nice to put a little more space between these, feels like proximity is awkward for which icon is with which number

<img width="434" alt="Screenshot 2025-02-28 at 2 42 40 PM" src="https://github.com/user-attachments/assets/66a36217-d19c-4ea5-b3c0-2f0d61cf9912" />